srcutil/code_style.py

```diff
@@ -106,7 +106,7 @@
     rv = po.wait()
     if rv != 0:
         print('Warning: Could not analyze {}'.format(f))
-    count = len([line for line in output.split('\n') if line])
+    count = len([line for line in output.decode("utf-8").split('\n') if line])
```

Comment:
Is this part of the bug or just an opportunity to fix?

Reply:
It's just to make it compatible to python 3, I guess you do not install our prehook that check code style..